apps/sap-commerce-cloud/src/api/fetchProductPreviews.ts

```diff
@@ -23,14 +23,9 @@ export async function fetchProductPreviews(
         },
       };
   for (const sku of skus) {
-    const splitSku = sku.split(':');
-    const baseSite = splitSku[0];
-    const skuId = splitSku[1];
-    skuIds.push(skuId);
+    skuIds.push(sku.split('/products/').pop()!);
     const response = await fetch(
-      parameters.installation.apiEndpoint +
-        `/occ/v2/${baseSite}/products/` +
```

Comment:
Is there a reason to remove the `parameters.installation.apiEndpoint + '/occ/etc`? This looks to be causing an issue when the Product Previews are fetched in the Field: 
<img width="577" alt="Screenshot 2023-04-03 at 1 20 54 PM" src="https://user-images.githubusercontent.com/122041643/229606609-2260cced-32e2-4715-ab96-bea733640a24.png">

Reply:
Convo was over slack, this was due to old data being in the blogpost